warehouse/ingest-core/src/main/java/datawave/ingest/config/BaseHdfsFileCacheUtil.java

```diff
@@ -17,7 +17,8 @@ public abstract class BaseHdfsFileCacheUtil {
     protected Path cacheFilePath;
     protected final Configuration conf;
     protected AccumuloHelper accumuloHelper;
-    private static String delimiter = "\t";
+    protected String delimiter = "\t";
+    protected short cacheReplicas = 3;
```

Comment:
cacheReplicas only used in TableConfigCache.  Perhaps it should be moved there

Reply:
I figured we would want to use this for other types of HDFS file caches in future, but wasn't sure if implementing it in all of them was too much scope creep.